### DISCUSSION TITLE
Can we add Context menu/Right click menu in builder tool

### DISCUSSION THREAD

**Discussion by usmanqamar-int** [GrapesJS/grapesjs#4915] [Q&A]
Like this:
![image](https://user-images.githubusercontent.com/57623938/217273585-0ded3234-87da-42bf-a5bb-a84717a83533.png)

Any help would be appreciated

**Comment by artf** [comment]
I guess this could refer to https://github.com/GrapesJS/grapesjs/discussions/4844

**Reply by usmanqamar-int** [reply]
Thanks for your response, 
I have implemented the context menu.

**Reply by badiuciprian** [reply]
Hi, have you managed to open the context menu on rightclick on the selected component only? because I tried it and the context was opening anywhere on the canvas
